bin/commands.js

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env node
+const chalk = require('chalk');
+const got = require('got')
+
+const { backupMessages } = require("./backupMessages");
+
+const generateRandomNumber = Math.floor(Math.random() * (backupMessages.length - 1));
```

Comment:
`const generateRandomNumber = Math.floor(Math.random() * (backupMessages.length - 1));`

This could be moved to a helper file

Reply:
[2e868a5dc48eb882ab144d69dbebb79a9e9766e1](https://github.com/ellie-keen/lolcommit/pull/11/commits/2e868a5dc48eb882ab144d69dbebb79a9e9766e1)